Components/Alert/alert.css

```diff
@@ -0,0 +1,58 @@
+.alert {
+  margin: 0.625rem;
+  padding: 0.938rem;
+  display: flex;
```

Comment:
you can use utility for this

Reply:
I have used class alert as a utility in the whole HTML structure where it contains flex so I didn't include it as an extra utility in the HTML structure.